packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts

```diff
@@ -183,7 +183,7 @@ export default function setAlertText(alert: any, callback: any) {
                 alert.params.payoutNumerators
               );
               const outcomeDescription =
-                outcome === null
+                outcome === MALFORMED_OUTCOME
```

Comment:
malformed and invalid are different concepts. we'd need to use the new IsInvalid property here